webapp/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,53 @@
+import './Avatar.css';
+
+import { Icon, IconVariant, IconSize } from '../index';
+import { Color } from '../../../shared/types';
+
+type AvatarStatus = 'online' | 'offline' | 'playing';
+
+type AvatarProps = {
+  url: string;
+  status?: AvatarStatus;
+};
+
+type LargeAvatarProps = AvatarProps & {
+  edit?: boolean;
+  caption?: string;
+};
+
+export function SmallAvatar({ url, status }: AvatarProps) {
+  return (
+    <div className={`avatar-small__image-wrapper  avatar-status--${status}`}>
```

Comment:
```suggestion
  const statusClassName = status ? `avatar-status--${status}` : '';

  return (
    <div className={`avatar-small__image-wrapper ${statusClassName}`}>
```

The status is `undefined` when not set and adds a class of `avatar-status--undefined`

What do you think about setting it to an empty string in this case?

Reply:
Yes, good catch, I'll fix it :ok_hand: